src/mousetracker.js

```diff
@@ -2959,7 +2959,7 @@
         }
 
         // OS-specific gestures (e.g. swipe up with four fingers in iPadOS 13)
-        if (gPoints[ 0 ].type === "touch" && typeof gPoints[ 0 ].currentPos === "undefined") {
+        if (typeof gPoints[ 0 ].currentPos === "undefined") {
```

Comment:
Interesting. Out of curiosity, what is the `type` in this situation?

Also, should it really be the string `"undefined"`, or the value `undefined`?

Reply:
Okay, good to know, regarding the pen type.

And sure enough, it's a `typeof`... don't know why I didn't notice that before!